This is a testharness.js-based test.
PASS Service Worker: Cache.add and Cache.addAll 
FAIL Cache.add called with no arguments undefined is not a function(stack: TypeError: undefined is not a function
    at http://127.0.0.1:8000/serviceworker/resources/cache-add-worker.js:6:13)
FAIL Cache.add called with relative URL specified as a string undefined is not a function(stack: TypeError: undefined is not a function
    at http://127.0.0.1:8000/serviceworker/resources/cache-add-worker.js:12:18)
FAIL Cache.add called with non-HTTP/HTTPS URL undefined is not a function(stack: TypeError: undefined is not a function
    at http://127.0.0.1:8000/serviceworker/resources/cache-add-worker.js:21:13)
FAIL Cache.add called with Request object undefined is not a function(stack: TypeError: undefined is not a function
    at http://127.0.0.1:8000/serviceworker/resources/cache-add-worker.js:28:18)
FAIL Cache.add called with Request object with a used body undefined is not a function(stack: TypeError: undefined is not a function
    at http://127.0.0.1:8000/serviceworker/resources/cache-add-worker.js:43:19)
FAIL Cache.add called twice with the same Request object undefined is not a function(stack: TypeError: undefined is not a function
    at http://127.0.0.1:8000/serviceworker/resources/cache-add-worker.js:52:18)
FAIL Cache.add with request that results in a status of 404 undefined is not a function(stack: TypeError: undefined is not a function
    at http://127.0.0.1:8000/serviceworker/resources/cache-add-worker.js:66:18)
FAIL Cache.add with request that results in a status of 500 undefined is not a function(stack: TypeError: undefined is not a function
    at http://127.0.0.1:8000/serviceworker/resources/cache-add-worker.js:74:18)
FAIL Cache.addAll with no arguments undefined is not a function(stack: TypeError: undefined is not a function
    at http://127.0.0.1:8000/serviceworker/resources/cache-add-worker.js:83:13)
FAIL Cache.addAll with a mix of valid and undefined arguments undefined is not a function(stack: TypeError: undefined is not a function
    at http://127.0.0.1:8000/serviceworker/resources/cache-add-worker.js:93:13)
FAIL Cache.addAll with string URL arguments undefined is not a function(stack: TypeError: undefined is not a function
    at http://127.0.0.1:8000/serviceworker/resources/cache-add-worker.js:102:18)
FAIL Cache.addAll with Request arguments undefined is not a function(stack: TypeError: undefined is not a function
    at http://127.0.0.1:8000/serviceworker/resources/cache-add-worker.js:116:18)
FAIL Cache.addAll with a mix of succeeding and failing requests undefined is not a function(stack: TypeError: undefined is not a function
    at http://127.0.0.1:8000/serviceworker/resources/cache-add-worker.js:131:18)
FAIL Cache.addAll called with the same Request object specified twice undefined is not a function(stack: TypeError: undefined is not a function
    at http://127.0.0.1:8000/serviceworker/resources/cache-add-worker.js:142:13)
Harness: the test ran to completion.

